src/PowerShellEditorServices/Services/Symbols/SymbolType.cs

```diff
@@ -41,6 +43,74 @@ internal enum SymbolType
         /// <summary>
         /// The symbol is a hashtable key
         /// </summary>
-        HashtableKey
+        HashtableKey,
+
+        /// <summary>
+        /// The symbol is a class
+        /// </summary>
+        Class,
+
+        /// <summary>
+        /// The symbol is a enum
```

Comment:
does that mean `[SomeEnum]` or does it mean `enum SomeEnum`?

Reply:
`enum SomeEnum` since `[SomeEnum]` is a type constraint (and so is a `SymbolType.Type`).